src/components/PhotoLibrary/PhotoGalleryImage.js

```diff
@@ -0,0 +1,85 @@
+// @flow
+
+import type { Node } from "react";
+import React from "react";
+import {
+  Image, Pressable
+} from "react-native";
+import Icon from "react-native-vector-icons/MaterialCommunityIcons";
+import { useBetween } from "use-between";
+
+import colors from "../../styles/colors";
+import { imageStyles } from "../../styles/photoLibrary/photoGallery";
+import useSelectedPhotos from "./hooks/useSelectedPhotos";
+
+const MAX_PHOTOS_ALLOWED = 20;
+
+type Props = {
+  item: Object,
+  setShowAlert: Function,
+  selectedAlbum: string,
+  getAllPhotos: Function,
+  updatePhotoGallery: Function
+}
+
+const PhotoGalleryImage = ( {
+  item,
+  setShowAlert,
+  selectedAlbum,
+  getAllPhotos,
+  updatePhotoGallery
+}: Props ): Node => {
+  const useSharedSelectedPhotos = ( ) => useBetween( useSelectedPhotos );
+  const {
+    selectedPhotos,
+    unselectPhoto,
+    selectPhoto
+  } = useSharedSelectedPhotos( );
```

Comment:
This seems like an excessive use of `useBetween`, or any way to share state. A PhotoGalleryImage component should presumably concern itself with a single image. `isSelected` makes sense as a prop here, but keeping track of which images are selected seems like a concern for whatever component contains all the images (which is probably `<PhotoGallery>` here).

Reply:
That's fair, I will change this.